src/backend/proxy.ts

```diff
@@ -25,18 +24,18 @@ const restream = (proxyReq: ClientRequest, req: IncomingMessage) => {
     }
 };
 
-export const doProxy = (context: string, applicationName: ApplicationName): RequestHandler => {
-    return createProxyMiddleware(context, {
+export const doProxy = (
+    applicationName: ApplicationName,
+    pathRewrite?: { [regexp: string]: string }
+): RequestHandler => {
+    return createProxyMiddleware({
+        target: `${miljø.clients[applicationName].url}`,
         changeOrigin: true,
-        logLevel: 'info',
-        logProvider: () => {
-            return logger;
-        },
-        onProxyReq: restream,
-        pathRewrite: (path: string) => {
-            return path.replace(context, '');
-        },
         secure: true,
-        target: `${miljø.clients[applicationName].url}`,
+        logger: console,
+        on: {
+            proxyReq: restream,
+        },
+        pathRewrite: pathRewrite,
```

Comment:
For meg virker det om jeg sletter pathRewrite også. Trenger vi den då? 👀

Reply:
Mulig den var unødvendig tidligere og.. 🥩

Reply:
Risikoen ved å ha den her er hvis man har dobbel path - da fjerner den "for mye":

`/dokument/dokument/noe` => `/noe` istedenfor `/dokument/noe` 😱 
Dette small i EF :D

Reply:
Nice! Da fjerner jeg `pathRewrite`